Userland/Utilities/markdown-check.cpp

```diff
@@ -233,14 +234,22 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     outln("Reading and parsing Markdown files ...");
     HashMap<String, MarkdownLinkage> files;
     for (auto path : file_paths) {
-        auto file_or_error = Core::File::open(path, Core::OpenMode::ReadOnly);
+        auto file_or_error = Core::Stream::File::open(path, Core::Stream::OpenMode::Read);
         if (file_or_error.is_error()) {
-            warnln("Failed to read {}: {}", path, file_or_error.error());
+            warnln("Failed to open {}: {}", path, file_or_error.error());
             // Since this should never happen anyway, fail early.
             return file_or_error.release_error();
         }
         auto file = file_or_error.release_value();
-        auto content_buffer = file->read_all();
+
+        auto content_buffer_or_error = file->read_all();
```

Comment:
doesn't `TRY(file->read_all())` work here?

Reply:
But you're returning if there's an error. Since when does that carry on to the next file? 🤨

Reply:
Ohhhh you're right. :man_facepalming:

I think I wrote it like this to match the original error handling, which prints an error saying which file had the problem. That's useful when it's dealing with several files. Hopefully we'll figure out how to do that with ErrorOr eventually.

Sorry if I've seemed a bit dismissive of your feedback - my intention here was to quickly port things to Core::Stream without doing any other refactoring or fixes. It's true most of these could be modified to be better, but then this PR would grow even more chonky. :sweat_smile: It's already big enough to hang around quite a while.